packages/paste-core/components/paragraph/__tests__/paragraph.test.tsx

```diff
@@ -1,58 +1,20 @@
 import * as React from 'react';
-import renderer from 'react-test-renderer';
-import {render} from 'react-dom';
+import {render as testRender} from '@testing-library/react';
 import {axe} from 'jest-axe';
-import {Theme} from '@twilio-paste/theme';
 import {Paragraph} from '../src';
 
-describe('Paragraph', () => {
-  it('it should render a single paragraph', (): void => {
-    const tree = renderer
-      .create(
-        <Theme.Provider theme="console">
-          <Paragraph>This is a paragraph</Paragraph>
-        </Theme.Provider>
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
+describe('General', () => {
+  it('should render', (): void => {
+    const textContent = `This is a paragraph`;
+    const {getByText} = testRender(<Paragraph>{textContent}</Paragraph>);
+    expect(getByText(textContent)).toBeDefined();
   });
+});
 
-  it('should have no accessibility violations', async () => {
-    const container = document.createElement('div');
-    document.body.append(container);
-    render(
-      <Theme.Provider theme="console">
-        <Paragraph>This is a paragraph</Paragraph>
-      </Theme.Provider>,
-      container
-    );
-    const results = await axe(document.body);
+describe('Accessibility', () => {
+  it('Should have no accessibility violations', async () => {
+    const {container} = testRender(<Paragraph>Hello world!</Paragraph>);
```

Comment:
I noticed that we were [doing quite a bit in other tests](https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/form/__tests__/checkbox.test.tsx#L194), but I _think_ this just works?

Reply:
Yup, I've switched [to this in other components](https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/menu/__tests__/index.spec.tsx#L111), still learning the ropes of testing library